backends/events/sdl/sdl-events.cpp

```diff
@@ -516,7 +559,7 @@ bool SdlEventSource::handleKeyUp(SDL_Event &ev, Common::Event &event) {
 
 	event.type = Common::EVENT_KEYUP;
 	event.kbd.keycode = SDLToOSystemKeycode(ev.key.keysym.sym);
-	event.kbd.ascii = mapKey(ev.key.keysym.sym, (SDLMod)ev.key.keysym.mod, (Uint16)ev.key.keysym.unicode);
+	event.kbd.ascii = mapKey(ev.key.keysym.sym, (SDLMod)ev.key.keysym.mod, 0);
```

Comment:
Please, see the discussion about that on #540 and #545. Here's a tl;dr to save you some time:
* Since the unicode value for key up events is always 0 there's no differences. We still set some ASCII values based on key codes.